lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -762,6 +762,9 @@ defmodule ExUnit.Assertions do
   Asserts `expression` will exit.
   Returns the exit status/message or fails otherwise.
 
+  catch_exit is about catching an exit raised by the current process itself.
```

Comment:
I would include the arity when mentioning functions (that’s how usually functions are identified in Elixir) and backticks (to display them as code and autolinking): `catch_exit/1` and `assert_received/2`.

In addition, I would also use backticks with the code `Process.flag(:trap_exit, true)`; and there is a typo `s/stared/started/` :)

Reply:
Also, what do you think about giving a more complete example:

```elixir
Process.flag(:trap_exit, true)
pid = spawn_link(fn -> Process.exit(self(), :normal) end)
assert_receive {:EXIT, ^pid, :normal}
```